src/components/HallOfFameButton.js

```diff
@@ -0,0 +1,20 @@
+const ManuButton = (firstLetterButtonInnerText, restOfTextButtonInnerText) => {
```

Comment:
Literówka oraz przydałoby się nazwać lepiej propsy
```suggestion
const MenuButton = (firstLetter, restOfText) => {
```

Reply:
tak próbowałem na początku, ale w metodzie tworzę nowe zmienne z takimi nazwami, dlatego takie wydłużone nazwy.
myślisz, że to się nie rozjedzie przy kompilacji kodu jak nazwę przyjmowane zmienne tak samo jak nowe zmienne w metodzie?

Reply:
Masz rację

Reply:
Popraw tylko literówkę i oznacza konwersacje jako rozwiązana